crates/sozo/src/init.rs

```diff
@@ -38,7 +38,14 @@ pub fn run(args: InitArgs) {
         None => current_dir().unwrap(),
     };
 
-    let template_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("sozo-template");
+    let template_dir = if let Ok(cargo_manifest_dir) = var("CARGO_MANIFEST_DIR") {
+        PathBuf::from(cargo_manifest_dir).join("sozo-template")
+    } else {
+        let mut binary_dir = current_exe().unwrap();
```

Comment:
Hmm are you sure this will work? I don't think `template-dir` will exist for the distributed binary. I think we will have to either embed these files in the binary or use git repo and clone it. I feel like the latter is probably the better solution

Reply:
true - that was an oversight. Perhaps we make it just clone the dojo-starter (which i will update to the sozo-template)

Reply:
cool yeah i think that works well